spec/models/concerns/mobile_workflow/displayable/steps/form_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
```

Comment:
I think it would be better not to load Rails in these kind of tests if these modules are intended to be loaded in plain Ruby classes. Loading Rails allows mistakenly use Rails methods and tests would pass anyway.

Reply:
Yes @pjmartorell , I can create a separate PR with tests for those utilities intended to be used also without Rails.

Reply:
This is the [PR](https://github.com/FutureWorkshops/mobile_workflow/pull/21) that tests those methods with and without Rails.